### PR TITLE
🪣 `ci asset`

### PR DESCRIPTION
**Pull request by mrharpo**
# `ci asset`
- Adds CLI for asset search
- Includes test

### CODE REVIEW

File: tests/sonyci/asset.toml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[sonyci]
+asset = '554544ceaf6b4c94a4a06cee5bc1f39f'
```
Comment (afred) [inline tests/sonyci/asset.toml:2]: is this used? Seems redundant with the `asset_id` fixture

Comment (mrharpo) [reply tests/sonyci/asset.toml:2]: It's a real asset in the `My Workspace` test workspace:

`cpb-aacip-e4308199588.txt`

> hello people!!!

Comment (afred) [reply tests/sonyci/asset.toml:2]: right, but we can get rid of the toml file right, and just use the fixture. That's all i can see being used.

**Review by afred — APPROVED** [review]
👍

**Pull request merged**